wp-cli/vip-migrations.php

```diff
@@ -147,6 +147,54 @@ public function validate_attachments( $args, $assoc_args ) {
 		WP_CLI\Utils\write_csv( $file_descriptor, $output );
 		fclose( $file_descriptor );
 	}
+
+	/**
+	 * Import user meta attributes from a CSV file.
+	 *
+	 * @subcommand import-user-meta
+	 * @synopsis <file> [--user_key=<userlogin>]
```

Comment:
We need more docs here.

What is `user_key`? What structure should the csv have?